### PR TITLE
Various form stuff

### PR DESCRIPTION
**Pull request by MrMaksimize**
Bringing in some fixes for choices, schema and fields.  setting up field values to work properly.

@chrisrodz  there are still issues with the other fields.  Here's what needs to happen for those.

When it's a single select / radio field, the other should fully override the value.

When it's a multiple select / checkboxes field, the other should get appended to the array getting stored.

In the edit views, other fields are not properly handled as well as some other fields.  They don't get their default values set properly from what it's in the db.  I'm reviewing now. 

HOLD

### CODE REVIEW

**Comment by chrisrodz** [comment]
When it's a single field I tested in and the behavior is exactly as you mentioned. At least on the last commit I pushed.

On multiple select fields it gets appended but the 'other' checkbox also gets it's text appended because of previous behavior. This need more fixing.

**Comment by MrMaksimize** [comment]
@chrisrodz still needs a test to make sure it works, but i'm too damn tired :)  

Please review when u can :)  @ainsleywagon would love if you looked as well :)

File: app.js [hunk 1]
```diff
@@ -74,6 +74,8 @@ app.use(methodOverride());
 app.use(cookieParser());
 app.use(session({
   secret: secrets.sessionSecret,
+  saveUninitialized: true,
```
Comment (MrMaksimize) [inline app.js:77]: I'm debating this

**Comment by ainsleywagon** [comment]
Okay! I'll checky when I get home 😊

**Comment by ainsleywagon** [comment]
@MrMaksimize Can you add a screenshot?

**Comment by chrisrodz** [comment]
No templates where modified, so the webpage itself looks the same.

**Pull request merged**